database_admin/migrations/105_applicable_advisories_count.up.sql

```diff
@@ -0,0 +1,71 @@
+ALTER TABLE system_platform ADD COLUMN installable_advisory_count_cache     INT NOT NULL DEFAULT 0;
+ALTER TABLE system_platform ADD COLUMN installable_advisory_enh_count_cache INT NOT NULL DEFAULT 0;
+ALTER TABLE system_platform ADD COLUMN installable_advisory_bug_count_cache INT NOT NULL DEFAULT 0;
+ALTER TABLE system_platform ADD COLUMN installable_advisory_sec_count_cache INT NOT NULL DEFAULT 0;
+ALTER TABLE system_platform ADD COLUMN applicable_advisory_count_cache     INT NOT NULL DEFAULT 0;
+ALTER TABLE system_platform ADD COLUMN applicable_advisory_enh_count_cache INT NOT NULL DEFAULT 0;
+ALTER TABLE system_platform ADD COLUMN applicable_advisory_bug_count_cache INT NOT NULL DEFAULT 0;
+ALTER TABLE system_platform ADD COLUMN applicable_advisory_sec_count_cache INT NOT NULL DEFAULT 0;
+
```

Comment:
Same here
```
ALTER TABLE system_platform ADD COLUMN installable_advisory_count_cache     INT NOT NULL DEFAULT 0,
                            ADD COLUMN installable_advisory_enh_count_cache INT NOT NULL DEFAULT 0,
                            ADD COLUMN installable_advisory_bug_count_cache INT NOT NULL DEFAULT 0,
                            ADD COLUMN installable_advisory_sec_count_cache INT NOT NULL DEFAULT 0,
                            ADD COLUMN applicable_advisory_count_cache     INT NOT NULL DEFAULT 0,
                            ADD COLUMN applicable_advisory_enh_count_cache INT NOT NULL DEFAULT 0,
                            ADD COLUMN applicable_advisory_bug_count_cache INT NOT NULL DEFAULT 0,
                            ADD COLUMN applicable_advisory_sec_count_cache INT NOT NULL DEFAULT 0;
```

Reply:
Actually the old `advisory_*_count_cache` columns are now `installable_advisory_*_count_cache` so it's easier to rename them (will keep their value):
```
ALTER TABLE system_platform RENAME COLUMN advisory_count_cache TO installable_advisory_count_cache;
ALTER TABLE system_platform RENAME COLUMN advisory_enh_count_cache TO installable_enh_advisory_count_cache;
ALTER TABLE system_platform RENAME COLUMN advisory_bug_count_cache TO installable_bug_advisory_count_cache;
ALTER TABLE system_platform RENAME COLUMN advisory_sec_count_cache TO installable_sec_advisory_count_cache;
ALTER TABLE system_platform ADD COLUMN applicable_advisory_count_cache     INT NOT NULL DEFAULT 0,
                            ADD COLUMN applicable_advisory_enh_count_cache INT NOT NULL DEFAULT 0,
                            ADD COLUMN applicable_advisory_bug_count_cache INT NOT NULL DEFAULT 0,
                            ADD COLUMN applicable_advisory_sec_count_cache INT NOT NULL DEFAULT 0;
```